examples/server/server.cpp

```diff
@@ -489,8 +489,13 @@ struct result_timings {
     double predicted_per_token_ms;
     double predicted_per_second;
 
+    // Optional speculative metrics - only included when > 0
+    int32_t draft_n = 0;
+    int32_t draft_accepted_n = 0;
```

Comment:
Be consistent with the names:

```suggestion
    int32_t draft_n_total = 0;
    int32_t draft_n_accepted = 0;
```